build.sbt

```diff
@@ -34,7 +29,7 @@ lazy val files = project
 
 lazy val mqtt = project
   .in(file("mqtt"))
-  .enablePlugins()
+  .enablePlugins(AutomateHeaderPlugin)
```

Comment:
It would be great if we could do it. But autoplugins can not enable other autoplugins.